### PR TITLE
Add a test for the upgrade step created in #191

### PR DESCRIPTION
**Pull request by pysailor**
refs #191

### CODE REVIEW

**Comment by pysailor** [comment]
@jenkins-plone-org please run jobs

**Comment by gforcada** [comment]
The 5.1 failure is probably due to missing the nogopip fix that @pbauer did, so a rebase is probably enough.

**Comment by pbauer** [comment]
Why are we running a test for 5.2a1 against 5.1? That can't work.

File: plone/app/upgrade/v52/tests.py [hunk 1]
```diff
@@ -11,10 +12,25 @@
...
+        from plone.app.upgrade.v52.alphas import rebuild_memberdata
+        rebuild_memberdata(portal)
+        tool = getToolByName(portal, 'portal_memberdata')
+        self.assertIn(
+            'test_user_1_', tool._members.keys())
+
+
 def test_suite():
     # Skip these tests on Plone 4
     if not PLONE_5:
```
Comment (pbauer) [inline plone/app/upgrade/v52/tests.py]: This needs to be changed so that this test only runs in 5.2

File: plone/app/upgrade/v52/tests.py [hunk 2]
```diff
@@ -11,10 +12,25 @@
     PLONE_5 = True
 
 
+class UpgradeMemberData51to512Test(unittest.TestCase):
```
Comment (pbauer) [inline plone/app/upgrade/v52/tests.py]: 51to512 should be 51to52

**Comment by pbauer** [comment]
@jenkins-plone-org please run jobs

**Comment by pysailor** [comment]
rebased on master...
@jenkins-plone-org please run jobs

**Comment by gforcada** [comment]
Could you add a news entry file and merge? 👍

**Comment by pysailor** [comment]
I'm actually not sure what to write into the news file. The actual PR #191 already contained a news entry. This one just extends the work done there by adding a test.

**Comment by gforcada** [comment]
Fair enough :+1:

**Pull request merged**